src/module/comment.js

```diff
@@ -0,0 +1,167 @@
+  const formHeader = document.createElement('h3');
+  const commentHeader = document.createElement('h3');
+
+  mealCard.className = 'mealCard';
+  mealCard.id = 'popMealCard';
+  allComments.className = 'allComments';
+
+  mealImg.setAttribute('src', meal.strMealThumb);
+  mealImg.setAttribute('alt', meal.strMeal);
+  mealImg.className = 'meal-img';
+
+  mealTitle.textContent = meal.strMeal;
+  mealTitle.className = 'meal-title';
+  mealTitle.id = 'meal-title';
+  mealVideoLink.id = 'video-link';
+
+  mealRecipe.innerHTML = meal.strInstructions;
+  mealRecipe.className = 'recipe';
+  mealRecipe.id = 'popup-meal-recipe';
+
+  mealVideoLink.setAttribute('href', meal.strYoutube);
+  mealVideoLink.textContent = 'Youtube Video'; // create a popup to play in-app rather that redirect (v2.0)
+
+  commentButton.textContent = 'Comments';
+  commentButton.className = 'comment';
+
+  form.setAttribute('method', 'post');
+  form.id = 'popup-form';
+  name.setAttribute('type', 'text');
+  name.id = 'username';
+  name.setAttribute('placeholder', 'Your name');
+
+  comment.setAttribute('name', 'comment');
+  comment.setAttribute('rows', '10');
+  comment.setAttribute('cols', '60');
+  comment.setAttribute('placeholder', 'Your Comment...');
+
+  formHeader.innerText = 'Add a comment';
+  form.className = 'comment-form';
+  formHeader.className = 'formHeader';
+  form.append(name, linebreak, comment, linebreak);
+
+  showAllComments(mealId).then((data) => {
+    if (data.message === `${mealId} not found.` || data.message === 'item_id\' not found.') {
+      commentHeader.innerText = 'This Recipe has no comments yet!\nAdd some comments!';
+    } else {
+      commentHeader.innerText = `Comments (${data.length})`;
+      data.forEach((comm) => {
+        const li = document.createElement('li');
+        li.style.listStyle = 'none';
+        li.style.margin = '0 32px';
+        li.style.color = 'darkbrown';
+        li.append(`${comm.creation_date} ${comm.username} ${comm.comment}`);
+        allComments.append(li);
+      });
+    }
+  });
+
+  commentButton.addEventListener('click', () => {
+    makeComment(name, comment, mealId);
+    showAllComments(mealId).then((data) => {
+      if (data.length === `${mealId} not found.` || data.message === 'item_id\' not found.') {
+        commentHeader.innerText = 'This Recipe has no comments yet!\nAdd some comments!';
+      } else {
+        commentHeader.innerText = `Comments (${data.length})`;
+        data.forEach((comm) => {
+          const li = document.createElement('li');
+          li.style.listStyle = 'none';
+          li.style.margin = '0 32px';
+          li.style.color = 'darkbrown';
+          li.append(`${comm.creation_date} ${comm.username} ${comm.comment}`);
+          allComments.append(li);
+        });
+      }
+    });
+  });
+
+  mealCard.append(
+    mealImg,
+    mealTitle,
+    mealRecipe,
+    mealVideoLink,
+    commentHeader,
+    // li,
```

Comment:
- [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.